src/Aspire.Dashboard/Model/ResourceMenuItems.cs

```diff
@@ -149,6 +149,7 @@ public static void AddMenuItems(
                     Text = url.Text,
                     Tooltip = url.Url,
                     Icon = s_linkIcon,
+                    Class = "resource-actions-url-menuitem",
```

Comment:
Is there a reason to not apply the class to all menu items at a lower level?

Reply:
For example, I imagine this problem could also happen if there is a very long command name.

Reply:
No reason, added to all fluent-menu-items.